Stride.BepuPhysics.Navigation/StrideGeomProvider.cs

```diff
@@ -47,44 +44,12 @@ public RcTriMesh GetMesh()
 
 	public RcVec3f GetMeshBoundsMin()
 	{
-		return _boundsmin;
+		return _bmin;
 	}
 
 	public RcVec3f GetMeshBoundsMax()
 	{
-		return _boundsmax;
-	}
-
-	public void CalculateNormals()
```

Comment:
I think this may have been needed due to the left vs right handed design. I had to convert the indices to Left handed and then I believe this was needed as well or else the faces would be pointing the wrong direction. We can leave it out for now though and I will have to get the demo scene wroking again to properly test.

Reply:
Ah I see, I didn't find anything that referenced this method and since it wasn't that great perf wise I thought it best to remove it entirely instead of improving it.
Although normals would be for debugging then, I'm not too sure what uses there is for normals in computing navigation ? Best to avoid computing/storing them if possible, there's already a fair amount of duplicate mesh data being allocated between the rendering, physics and navigation. All of those may end up taking a lot of our ram.